e2e/johtoselvitys_tilaus_kasittely.spec.ts

```diff
@@ -0,0 +1,136 @@
+import { test, expect } from '@playwright/test';
+import { perustaja, vastaava, suorittaja, rakennuttaja, asianhoitaja, testiData, testiOsoite, } from './_setup';
+
+test.beforeAll('Log testdata', async ({ }) => {
```

Comment:
SonarCloud on merkinnyt tämän bugiksi: https://sonarcloud.io/project/issues?severities=BLOCKER%2CCRITICAL%2CMAJOR%2CMINOR&sinceLeakPeriod=true&issueStatuses=OPEN%2CCONFIRMED&types=BUG&pullRequest=1021&id=City-of-Helsinki_haitaton-ui&open=AZPeu5oO12lOrSoaObTU&tab=code

async({ }): "Unexpected empty object pattern."